### PR TITLE
Created preprocessing script for French version of legislation from Algeria and preprocessed legislation

### LINKED ISSUE #33: Develop preprocessing script for legislation from Algeria

### PR DESCRIPTION
**Pull request by danpdraper**
Closes #33

### CODE REVIEW

File: raw_data/preprocessed/africa/algeria_french.txt [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+(98) Pour certaines catégories de déchets visées à l'article 93 et précisées par décret, le ministre chargé de l'environnement fixe , sur tout ou partie du territoire national, les conditions d'exercice de l'activité d'élimination telle qu'elle est définie à l'article 93 de la présente loi et en particulier celles de transporteur de déchets. Ces mêmes catégories de déchets ne peuvent être traitées que dans les installations prévues ou agréées. Un décret pris sur le rapport du ministre chargé de l'environnement et les ministres concernés fixe les modalités d'application du présent article.
```
Comment (Skyrolon) [inline raw_data/preprocessed/africa/algeria_french.txt]: There is an unnecessary space between "fixe" and the following ",".

File: raw_data/preprocessed/africa/algeria_french.txt [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
...cidences sur le milieu naturel, peuvent porter atteinte à ce dernier, doivent comporter une étude d'impact permettant d'en apprécier les conséquences. Un décret pris sur le rapport du département ministériel chargé de l'environnement et des ministres concernés précise les modalités d'application du présent article. Il fixe notamment: [•] d'une part, les conditions dans lesquelles les préoccupations d'environnement sont prises en compte dans les procédures réglementaires existantes; [•] d'autre part, le contenu de l'étude d'impact qui comprend au minimum une analyse de l'état initial du site et de son environnement, l'étude des modifications que le projet y engendrerait et les mesures envisagées pour supprimer, réduire et, si possible, compenser les conséquences dommageables pour l'environnement; [•] les conditions dans lesquelles l'étude d'impact sera rendue publique; [•] la liste limitative des ouvrages qui, en raison de la faiblesse de leurs répercussions sur l'environnement, ne sont pas soumis à la procédure de l'étude d'impact. Il fixe également les conditions dans lesquelles le ministre chargé de l'environnement pourra se saisir ou être saisi pour avis de toute étude d'impact.
```
Comment (Skyrolon) [inline raw_data/preprocessed/africa/algeria_french.txt:301]: Not sure if there should be another bullet point preceding "Il fixe également". It looks like that one from the formatting in the original, a line break and an indent, but I'm not sure what the French actually says.

File: raw_data/preprocessed/africa/algeria_french.txt (hunk 1 above)
Comment (danpdraper) [reply raw_data/preprocessed/africa/algeria_french.txt]: Good catch. I will update.

File: raw_data/preprocessed/africa/algeria_french.txt (hunk 2 above)
Comment (danpdraper) [reply raw_data/preprocessed/africa/algeria_french.txt:301]: "Il fixe également" should mark the beginning of a new sentence in this case. The formatting in the original is misleading. Although only "d'une part" and "d'autre part" are preceded by bullets therein, the wording is such that "Les conditions" and "La liste limitative" should also be preceded by bullets.

Comment (Skyrolon) [reply raw_data/preprocessed/africa/algeria_french.txt:301]: Understood, thank you for explaining.

**Pull request merged**